app/docs/md/routing/api-routes/index.md

```diff
@@ -76,3 +76,69 @@ API routes always return a `response` object:
 | `headers`      | `object` | Set response headers as key/value pairs
 | `session`      | `object` | Writes passed object to the session
 
+## Requesting Data Example
+
+Perhaps you would like to progressively enhances your web component to avoid a full page reload when requesting data from an Enhance backend. Here is an example of using `fetch` in the browser to make that request.
+
+```javascript
+const response = await fetch('/', {
+  headers: { 'accept': 'application/json' },
+  method: 'get'
+})
+
+const json = await res.json()
+// json: { favorites: ['coffee crisp', 'smarties'] }
+```
+
+## Sending Data Example
+
+Conversely, you may want to avoid doing a form post to prevent a full page reload in the browser. In order send data to an Enhance back end you must properly set the `Content-Type` header.
```

Comment:
```suggestion
Data can also be posted to Enhance backends via the client to avoid full page reloads.

In order to send data to an Enhance back end, you must properly set the `Content-Type` and `accept` headers. The `Content-Type` header informs the API route about the type of data that’s being sent, while the `accept` header instructs the API route as to what type of data to return.
```